### DISCUSSION TITLE
Buttons in alerts

### DISCUSSION THREAD

**Discussion by corinne-nagel** [twilio-labs/paste#1526] [Ideas] [answered]
Hi Paste homies 👋 

Angela and I were talking about putting a button in an [alert](https://paste.twilio.design/components/alert/) but it says "Avoid using buttons in alerts unless it triggers an action."

If we wooooould like it to trigger an action, maybe we can update the language to say something about allowing buttons in alerts? Anchors won't really work for an action so having that included somewhere might clear things up a bit.

Was thinking something like this?

![Screen Shot 2021-06-03 at 3 24 31 PM](https://user-images.githubusercontent.com/62260820/120713939-cfbb9580-c47f-11eb-9b03-1841a8d41ce5.png)

Also, not sure where this would be in your roadmap (maybe callout?), but it could be related to more of an in-page banner that we discussed like this:

![Screen Shot 2021-06-03 at 3 29 07 PM](https://user-images.githubusercontent.com/62260820/120714460-756f0480-c480-11eb-8a33-758dea1fde2a.png)

Let me know your thoughts! Down to work with you to contribute as well 😄

**Comment by kate-porter** [comment, marked as answer]
Oh heyyyy! So sorry for the delay on this!

Good call on updating the language on the Alert docs—I can go ahead submit a PR to do that. Sarah and I were just chatting about this, and we were thinking it might make most sense to just get rid of that do/don't on the Alert page about not using buttons, since we have a pattern around anchor vs. button usage now that covers that. What do you think?

And on your second question, yeah, that's probably a callout. It's not too high up on the roadmap, because it's pretty easily composable with box/text, and a lot of folks are generally using cards to do this type of messaging right now. Definitely down for a contribution!
